quickstart/python/from-blob/python-client/main.py

```diff
@@ -44,13 +44,20 @@ def transcribe():
     # create an instance of the transcription api class
     transcription_api = cris_client.CustomSpeechTranscriptionsApi(api_client=client)
 
-    # Use base models for transcription. Comment this block if you are using a custom model.
-    # Note: you can specify additional transcription properties by passing a
-    # dictionary in the properties parameter. See
-    # https://docs.microsoft.com/azure/cognitive-services/speech-service/batch-transcription
+    # Specify transcription properties by passing a dict to the properties parameter. See
+    # https://docs.microsoft.com/azure/cognitive-services/speech-service/batch-transcription#configuration-properties
     # for supported parameters.
+    properties = {
+        # 'PunctuationMode': 'Automatic',
+        # 'ProfanityFilterMode': 'None',
+        # 'AddWordLevelTimestamps': 'true',
+        # 'AddDiarization': 'true'
+    }
```

Comment:
>} [](start = 4, length = 1)

what about the other, perhaps mention dem also?
TranscriptionResultsContainerUrl
AddSentiment